splunk_eventgen/__main__.py

```diff
@@ -1,25 +1,29 @@
-'''
```

Comment:
we'd better consult splunk legal team to come up with a standard file headers for our eventgen project code.
Usually it describes the copy right or any legal concerns.

Reply:
I updated some outdated info for this pr.